OxfordBodleian/Juel-Jensen/BDLaethd23.xml

```diff
@@ -116,8 +219,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </collation>
                                 <condition key="good">
                                     The manuscript is in a good state of preservation. The folios only show slight traces of use on the lower margins
-                                  and stains of various origin. Few tears are present, e.g <locus target="#4 #34 #39"/>.
-                                  The binding lacks of the spine and of the enbands, which have been removed to repair the sewing.
+                                  and stains of various origin. Few tears are present, e.g. <locus target="#4 #34 #39"/>.
+                                  The binding lacks of the spine and of the endbands, which have been removed to repair the sewing.
```

Comment:
"The manuscript lacks a leather spine and endbands which may have been removed to repair the sewing."